homeassistant/components/html5/__init__.py

```diff
@@ -1 +1,50 @@
 """The html5 component."""
+
+import logging
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_PLATFORM, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv, discovery
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DATA_HASS_CONFIG, DOMAIN
+from .issues import async_create_html5_issue
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM = Platform.NOTIFY
+CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the HTML5 push notification component."""
+    hass.data[DATA_HASS_CONFIG] = config
+    existing_config_entry = hass.config_entries.async_entries(DOMAIN)
+
+    # Iterate all entries for notify to only get HTML5
+    for entry in config.get(Platform.NOTIFY, {}):
+        if entry[CONF_PLATFORM] != DOMAIN:
+            continue
```

Comment:
Can we use the same entrypoint for importing as how we set up the YAML config?

Reply:
Not sure what this means. The notify platform calls some magic function that only sets up yaml here.

Reply:
No, the YAML previously got setup in the `async_get_service`, so we should start the import from there. Check `mastodon` for an example

Reply:
Okay I'll take a look at doing this.